### ISSUE TITLE
Improve and extend Merge for Organizations object

### ISSUE THREAD

**Issue by oayvazoglusim** [NCI-Agency/anet#2992]
As a super user / administrator, I want to be able to merge duplicates of Organizations

As a super user / administrator, I want to have more control on which fields are taken from which object, and possibly given the ability to enter a new value of a field while merging.

**Comment by midmarch** [comment]
Blocked by #3147.

**Comment by cemalettin-simsoft** [comment]
We can reuse the merge functionality that is implemented in #3147

**Comment by midmarch** [comment]
Is this feature really necessary?

**Comment by gjvoosten** [comment]
> Is this feature really necessary?

Functionality is not needed by the user.
